ocr.ipynb

```diff
@@ -0,0 +1,181 @@
+{
+ "cells": [
```

Comment:
Can you link the notebook to the tutorial section next time? So that I can also review what it looks like in the tutorial section